newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js

```diff
@@ -221,7 +221,16 @@ export default class LayerRenderer {
 
   getUnrotatedInstanceSize = (instance: gdInitialInstance) => {
     if (instance.hasCustomSize())
-      return [instance.getCustomWidth(), instance.getCustomHeight()];
+      return [
+        instance.getCustomWidth() ||
+          (this.renderedInstances[instance.ptr]
```

Comment:
maybe `this.renderedInstances[instance.ptr]` can be put inside a variable like `renderedInstance` as this is used multiple times below too